swarm/CHANGELOG.md

```diff
@@ -13,6 +77,10 @@
   documentation](https://docs.rs/libp2p/latest/libp2p/swarm/trait.NetworkBehaviour.html) for
   details.
 
+- Update dial address concurrency factor to `8`, thus dialing up to 8 addresses concurrently for a single connection attempt. See `Swarm::dial_concurrency_factor` and [PR 2741].
+
+- Update to `libp2p-core` `v0.35.0`.
+
```

Comment:
Can we add the changelog entry above so this doesn't come up in the diff?

Reply:
I ordered the entries by significance. I don't think users care much about the `libp2p-core` update. I do think users care about how to upgrade from `NetworkBehaviourEventProcess`. Does that reasoning make sense @thomaseizinger?